frontend/lib/Views/enpoint_list_view.dart

```diff
@@ -111,8 +111,8 @@ class _EndpointListViewState extends State<EndpointListView> {
             FutureBuilder<EndpointData>(
               future: widget.gateway.getEndpointData(
                 expansionPanelEndpoint.id,
-                1,
-                0,
+                null,
+                null,
```

Comment:
Co to za `null'e`? :eyes:

Reply:
no to jest pewnie offset i limit, moze ladniej je zrobic opcjonalne, ale imo up to u Kuba

Reply:
można, ale to był case który miał być poprawiony przez @zbrze, a został nieruszony, nie wiem czy to ten task tbh